### ISSUE TITLE
No support for blank lines

### ISSUE THREAD

**Issue by OtherCroissant** [smortex/yaml-sort#26]
Hi, Thanks for the fast fix on my previous issue!

Really started using the yaml-sorter with actual content so I stumble upon some problems now and then. Hope you dont mind ;)

When we have the following content: 

```
test:
  testtest: blob %{variable} bliep
  testblaat: |- A long piece of text content where we want to determine the newlines
    for the resulting content in the yml file itself by
    placing content on a new line in the yml file.

    and then the customer wants a newline in this content...
```

Above example results in a "unexpected content" error with yaml-sort.

Possible fix is to change the content to a double quoted string and than explicitly put in \n characters, but this is not optimal.

Could you look into the |- support when you have the time?

**Comment by smortex** [comment]
It looks like your input is not valid YAML on line 3:

```sh-session
romain@zappy /tmp % cat foo.yaml
test:
  testtest: blob %{variable} bliep
  testblaat: |- A long piece of text content where we want to determine the newlines
    for the resulting content in the yml file itself by
    placing content on a new line in the yml file.

    and then the customer wants a newline in this content...
romain@zappy /tmp % ruby -e 'require "yaml"; puts YAML.load_file("foo.yaml")'
/usr/local/lib/ruby/3.0/psych.rb:457:in `parse': (foo.yaml): did not find expected comment or line break while scanning a block scalar at line 3 column 14 (Psych::SyntaxError)
	from /usr/local/lib/ruby/3.0/psych.rb:457:in `parse_stream'
	from /usr/local/lib/ruby/3.0/psych.rb:391:in `parse'
	from /usr/local/lib/ruby/3.0/psych.rb:280:in `unsafe_load'
	from /usr/local/lib/ruby/3.0/psych.rb:583:in `block in unsafe_load_file'
	from /usr/local/lib/ruby/3.0/psych.rb:582:in `open'
	from /usr/local/lib/ruby/3.0/psych.rb:582:in `unsafe_load_file'
	from -e:1:in `<main>'
```

Yet, a blank line in a `|-` value fail unexpectedly on line 7:

```sh-session
romain@zappy ~/Projects/yaml-sort % cat /tmp/foo.yaml
test:
  testtest: blob %{variable} bliep
  testblaat: |-
    A long piece of text content where we want to determine the newlines
    for the resulting content in the yml file itself by
    placing content on a new line in the yml file.

    and then the customer wants a newline in this content...
romain@zappy ~/Projects/yaml-sort % bundle exec exe/yaml-sort /tmp/foo.yaml
/tmp/foo.yaml:9: unexpected content
    and then the customer wants a newline in this content...
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by smortex** [comment]
Same issue for values on multiple lines without `|-`.

**Comment by smortex** [comment]
Fixed in yaml-sort-2.2.3, thanks!
